website/source/docs/guides/deployment.html.md

```diff
@@ -29,33 +27,33 @@ The following instance configurations are recommended.
 |||||**Azure**: Standard\_D4\_v3, Standard\_D5\_v3|
 |||||**GCE**: n1-standard-32, n1-standard-64|
 
-The **small size** instance configuration is appropriate for most initial production deployments, or for development/testing environments. The large size is for production environments where there is a consistently high workload.
+The **small size** instance configuration is appropriate for most initial production deployments, or for development/testing environments. The large size is for production environments where there is a consistently high workload. Suggested instance types are provided for common platforms, but do refer to platform documentation for up-to-date instance types that align with the recommended resources.
 
-~> **NOTE** For high workloads, ensure that the disks support high IOPS to keep up with the high Raft log update rate.
+~> **NOTE** For large workloads, ensure that the disks support a high number of IOPS to keep up with the rapid Raft log update rate.
 
-## 1.1 Datacenter Design
+## Datacenter Design
 
-A Consul cluster (typically 3 or 5 servers plus client agents) may be deployed in a single physical datacenter or it may span multiple datacenters. For a large cluster with high runtime reads and writes, deploying servers in the same physical location improves performance. In cloud environments, a single datacenter may be deployed across multiple availability zones i.e. each server in a separate availability zone within a single EC2 instance. Consul also supports multi-datacenter deployments via two separate clusters joined by WAN links. In some cases, one may also deploy two or more Consul clusters in the same LAN environment.
+A Consul cluster (typically 3 or 5 servers plus client agents) may be deployed in a single physical datacenter or it may span multiple datacenters. For a large cluster with a high number of reads and writes, deploying servers in the same physical location improves performance. In cloud environments, a single datacenter may be deployed across multiple availability zones i.e. each server in a separate availability zone within a single EC2 instance. Consul also supports multi-datacenter deployments via two separate clusters joined by WAN links. In some cases, one may also deploy two or more Consul clusters in the same LAN environment.
```

Comment:
I think the first sentence here is ambiguous. 

> A Consul cluster (typically 3 or 5 servers plus client agents)

We should be consistent about a "Consul Cluster" being a whole logical multi-DC cluster. We are not consistent about this in docs today but should improve that - Armon's intent was always that datacenters were a locality-based optimisation/fault tolerance mechanism but that it was the global federation of a set of datacenters that formed the logical "cluster"

>  or it may span multiple datacenters

This is a bit confusing. We _never_ recommend Raft (i.e. one set of servers) spanning high-latency links (i.e. multiple physical datacenters).

The exception is "Availability Zones" which are separate "datacenters" in the sense that they are different physical facilities, but importantly they have very low latency direct (non-WAN) connectivity between them.

If it's not clear how to reword this I'm happy to have a go but trying to avoid getting to distracted for now :)

Reply:
An initial rephrasing is at c7c620aa2